hazelcast-sql/src/main/java/com/hazelcast/sql/impl/type/converter/Converter.java

```diff
@@ -69,24 +72,25 @@ public abstract class Converter implements Serializable {
     protected static final int ID_ROW = 25;
 
     private final int id;
-    private final QueryDataTypeFamily typeFamily;
-
-    private final boolean convertToBoolean;
-    private final boolean convertToTinyint;
-    private final boolean convertToSmallint;
-    private final boolean convertToInt;
-    private final boolean convertToBigint;
-    private final boolean convertToDecimal;
-    private final boolean convertToReal;
-    private final boolean convertToDouble;
-    private final boolean convertToVarchar;
-    private final boolean convertToDate;
-    private final boolean convertToTime;
-    private final boolean convertToTimestamp;
-    private final boolean convertToTimestampWithTimezone;
-    private final boolean convertToObject;
-    private final boolean convertToJson;
-    private final boolean convertToRow;
+    private final transient QueryDataTypeFamily typeFamily;
+
+    private final transient boolean convertToBoolean;
+    private final transient boolean convertToTinyint;
+    private final transient boolean convertToSmallint;
```

Comment:
this will change `serialVersionUID` but we should be fine if Java serialization is used only for Jet